### ISSUE TITLE
Add default profile to user json if it is not specified

### ISSUE THREAD

**Issue by Karl-WE** [microsoft/terminal#6216]
# Description of the new feature/enhancement

Current version generates two settings json files. A baseline that may not be edited (ctrl+settings) and a user locale one that is generated too, but which can be edited by a user. 

Both will have auto generated profiles for recognized products ex. PoSh 7, cmd etc. 

That's great. 

For all users that might not want to dig deep into all possible settings that a json can possibly hold it's not easy to understand without reading docs and knowing of the existence of the global settings json how to alter a default console / profile 

Again changing the default json isn't possible. 

Adding a default profile (console) 
manually can be quite some work. 

It is more prone to user errors when setting this up or copy and adding from the default json. Especially json formatting issues without using a proper editor like VSC. Not all might want to install VSC or other code aware editors as notepad++ but by default we have Microsoft Notepad (app). 

Currently the local user json is missing this the line for the default profile. 

Mind Terminal users aren't necessarily Devs. #NotADev 

# Proposed technical implementation details (optional)
Please while generating the user local json, parse if it does contain a line for a default profile 

Add default profile otherwise as in global settings json. 

If it does exist don't touch.

This would ease the usage for new users.

**Comment by zadjii-msft** [comment]
I'm 100% positive that `defaultProfile` is already a part of the json file we generate on first launch:

https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalApp/userDefaults.json#L11

We'll take that `%DEFAULT_PROFILE%` and replace it with the guid of Powershell Core or legacy powershell, but that line is definitely always present.

**Comment by Karl-WE** [comment]
Thanks for checking, I will recheck this in a VM an report back. Errare humanum est. Yet I had to add it in my config file. (Did upgrade to the current from a pre-release version via Microsoft Store).

**Comment by Karl-WE** [comment]
Hi @zadjii-msft I cannot explain but I had to add this line manually to override the json global default which would choose cmd
the UUID below is a PosH7 release that got added automatically. 

  "defaultProfile": "{574e775e-4f2a-5b96-ac1e-a2962a402336}",

Given what Dustin said my json is old see the other feature request I've made a mention. Perhaps this might be a doable / good solution. 

I will close this for now, as I don't think I can repro this. My oldest backup does not contain the file before I added the lines.
